src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -161,7 +172,7 @@ public void setPerson(Person target, Person editedPerson) {
      */
     @Override
     public ObservableList<Person> getFilteredPersonList() {
-        return filteredPersons;
+        return sortedPriority;
```

Comment:
Is this supposed to be a new method instead of changing the old `getFilteredPersonList()` method?

Reply:
no because the other commands still uses this version of getFilteredPersonList() that still returns the correct list of person just that adding new emergency contact, change of priority level have to follow this sorted index.
![image](https://github.com/user-attachments/assets/482d4dcd-e9d4-4764-b0ce-0589ebfa8214)

`filteredPersons = new FilteredList<>(this.addressBook.getPersonList());` calls from UniquePersonList. The original list that is stored will still have features of adding/deleting commands as this sorted just touches UI side not the logical side.

Upon closing and reopening of app, it will revert to the non-sorted version while keeping all forms of added or deleted PersonList.